.mypy.ini

```diff
@@ -34,3 +34,13 @@ ignore_missing_imports = True
 
 [mypy-gunicorn.*]
 ignore_missing_imports = True
+
+[mypy-test_benchmarks_client_request]
+disable_error_code =
+  no-any-unimported,
+  misc
+
+[mypy-test_benchmarks_cookiejar]
+disable_error_code =
+  no-any-unimported,
+  misc
```

Comment:
It's not overly clear why we're disabling these here. Is it a case of being too noisy with pytest-codspeed untyped? In which case we can remove this once they've exposed type annotations.

Reply:
That’s exactly the reason.

Reply:
comments at in 19e6bd302fa43f617456f124c3a067e57b3a7d15